Tests/TestingTests/EventTests.swift

```diff
@@ -0,0 +1,68 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+import Foundation
+@testable
```

Comment:
Nit: our house style is to put these attributes on the same line as the `import` declaration.

Reply:
I did that because the line is then more than twice as long as the 80 width limit we seem to use, but changed.

Reply:
We only force the 80-column limit on documentation/comments, and that's because documentation gets word-wrapped when rendered and it helps give us a sense of how long paragraphs are. We don't demand that of code lines.